src/main/java/io/fabric8/vertx/maven/plugin/utils/MojoUtils.java

```diff
@@ -29,30 +29,37 @@
 import java.util.List;
 import java.util.Optional;
 
-import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.executionEnvironment;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.*;
 
 /**
  * @author kameshs
  */
 public class MojoUtils {
 
+    /*===  Plugin Keys ====*/
+
     public static final String JAR_PLUGIN_KEY = "org.apache.maven.plugins:maven-jar-plugin";
+    public static final String RESOURCES_PLUGIN_KEY = "org.apache.maven.plugins:maven-resources-plugin";
     public static final String VERTX_PACKAGE_PLUGIN_KEY = "io.fabric8:vertx-maven-plugin";
 
+    /*===  Plugins ====*/
 
     public static final String G_MAVEN_JAR_PLUGIN = "org.apache.maven.plugins";
     public static final String A_MAVEN_JAR_PLUGIN = "maven-jar-plugin";
     public static final String V_MAVEN_JAR_PLUGIN = "3.0.2";
 
+    public static final String G_MAVEN_RESOURCES_PLUGIN = "org.apache.maven.plugins";
+    public static final String A_MAVEN_RESOURCES_PLUGIN = "maven-resources-plugin";
+    public static final String V_MAVEN_RESOURCES_PLUGIN = "3.0.1";
```

Comment:
done! moved the version numbers to mojo property file called `vertx-maven-plugin.properties`, the MojoUtils class will load this property and will be used for version defaults